src/mame/casio/fp1100.cpp

```diff
@@ -713,10 +714,22 @@ ROM_START( fp1100 )
 	ROM_LOAD( "sub3.rom", 0x2000, 0xf80, BAD_DUMP CRC(fb2b577a) SHA1(a9ae6b03e06ea2f5db30dfd51ebf5aede01d9672))
 ROM_END
 
+// FP1000 has less memory than FP1100, using the same PCB, but with unpopulated sockets from RAM9 to RAM24 (only RAM1 to RAM8 are populated).
+ROM_START( fp1000 )
+	ROM_REGION( 0x9000, "ipl", ROMREGION_ERASEFF )
+	ROM_LOAD( "2l_a10_kkk_fp1000_basic.c1", 0x0000, 0x1000, CRC(9322dedd) SHA1(40a00684ced2b7ead53ca15a915d98f3fe00d3ba))
+
+	ROM_REGION( 0x3000, "sub_ipl", ROMREGION_ERASEFF )
+	ROM_LOAD( "jka_fp1000.e8",    0x0000, 0x1000, CRC(2aefa4e4) SHA1(b3cc5484426c19a7266d17ea5c4d55441b4e3be8))
+	ROM_LOAD( "jkc_fp1000.e21",   0x1000, 0x1000, CRC(67a668a9) SHA1(37fb9308505b47db36f8c341144ca3fe3fec64af))
+	ROM_LOAD( "upd7801g_118.bin", 0x2000, 0xf80, BAD_DUMP CRC(fb2b577a) SHA1(a9ae6b03e06ea2f5db30dfd51ebf5aede01d9672)) // Not dumped, borrowed from 'fp1100'
+ROM_END
+
 } // anonymous namespace
 
 
-/* Driver */
+// Drivers
 
-/*    YEAR  NAME    PARENT  COMPAT  MACHINE  INPUT   CLASS         INIT        COMPANY  FULLNAME   FLAGS */
+//    YEAR  NAME    PARENT  COMPAT  MACHINE  INPUT   CLASS         INIT        COMPANY  FULLNAME   FLAGS
```

Comment:
I dislike this cargo cult, it's outdated and it throws bad habits for people using this as a net safety for something they should look at where the macro is actually located.
Could you please get rid of it?

Reply:
Done!